packages/atomic/src/components/atomic-result-list/atomic-result-list.tsx

```diff
@@ -59,38 +56,43 @@ import {updateBreakpoints} from '../../utils/replace-breakpoint';
 })
 export class AtomicResultList implements InitializableComponent {
```

Comment:
Maybe both result list classes could implement a common interface
```suggestion
export class AtomicResultList implements InitializableComponent, AtomicBaseResultList {
```